x-pack/plugins/osquery/cypress/support/e2e.ts

```diff
@@ -78,8 +78,15 @@ Cypress.Commands.add(
   () => cy.get('body').click(0, 0) // 0,0 here are the x and y coordinates
 );
 
-Cypress.Commands.add('login', (role) => {
-  if (isServerless) {
+Cypress.Commands.add('login', (role, useCookiesForMKI = true) => {
```

Comment:
NIT: 
What do you think about reverting the logic of `useCookiesForMKI`: 

```Cypress.Commands.add('login', (role, doNotUseCookiesForMKI) => {```

and 

```if (isCloudServerless && doNotUseCookiesForMKI) {```

Then we do not need default value, and just pass `true` to the 'edge cases'. 
It seems more natural to me.
However not a biggie, it can also stay as it is - fine 👍